lib/pure/collections/lists.nim

```diff
@@ -178,6 +179,28 @@ proc newSinglyLinkedNode*[T](value: T): <//>(SinglyLinkedNode[T]) =
   new(result)
   result.value = value
 
+func toSinglyLinkedList*[T](elems: openArray[T]): SinglyLinkedList[T] {.since: (1, 6).} =
```

Comment:
`(1,5,1)`
otherwise you break devel code

(+ elsewhere)

Reply:
OK. So these will be later updated automatically to (1, 6)? (since these will actually only be in that version, at least according to the changelog :))

Reply:
no, no need to update anything, look at the implementation of `since`. It will "just work", because once 1.6 is released, `1.6 >= (1,5,1)`